velero/backup/common-service/label-common-service.sh

```diff
@@ -323,7 +326,7 @@ function label_subscription() {
     
     ${OC} label subscriptions.operators.coreos.com $cs_pm foundationservices.cloudpak.ibm.com=subscription -n $OPERATOR_NS --overwrite=true 2>/dev/null
     if [[ $ENABLE_CERT_MANAGER -eq 1 ]]; then
-        ${OC} label subscriptions.operators.coreos.com $cm_pm foundationservices.cloudpak.ibm.com=singleton-subscription -n $CERT_MANAGER_NAMESPACE --overwrite=true 2>/dev/null
```

Comment:
I have a couple thoughts on changing the label name:
1. we will need to update the instructions for OADP/Velero to make sure these are properly reflected
2. we need to include the cert manager crds (like certificate and issuer) because they are currently not included so when we restore cert-manager, we will fail to restore certs and issuers labeled cert-manager from the label-cert-manager.sh script

Its possible to lump all cert manager resources (including the subscription) together (I think) but we need to make sure CRDs are there and I don't seem them labeled in any of our existing scripts.